### ISSUE TITLE
Multiselect Popover with expanding input

### ISSUE THREAD

**Issue by qcharlieshi** [palantir/blueprint#2684]
#### Environment

- __Package version(s)__: 3.0.0
- __Browser and OS versions__: Chrome 67.0, Windows 10

#### Steps to reproduce
http://blueprintjs.com/docs/#select/multi-select
1. Select many items

#### Actual behavior

The popover stays in place causing it to cover up the last entered tag once it falls onto multiple lines

#### Expected behavior

The popover should adjust (I believe it does on scroll) to stay below the input

#### Possible solution

I haven't looked into this quite yet.

**Comment by giladgray** [comment]
yeah this is essentially a duplicate of #692. need to use `Popper#scheduleUpdate`.

**Comment by badams** [comment]
@giladgray This issue wasn't in 2.3.1, at least with multi-select see the attached recordings

[Blueprint 2 MultiSelect (Works as expected)](https://gfycat.com/JoyousClassicCockroach)
[Blueprint 3 MultiSelect (Not repositioning)](https://gfycat.com/EnergeticConfusedAmericanbittern)

Any idea when we can expect to be addressed? Its the only thing holding us back from rolling out our upgrade to 3.*
